3303Elevator/src/app/ElevatorSubsystem/ElevatorSubsystem.java

```diff
@@ -147,7 +110,29 @@ public void run(){
 				checkFloor(destFloor);
 			}
 			System.out.println("\n\n");
+		}*/
+	}
+	
+	public static void main(String[] args){
+		Config config = new Config("local.properties");
+		int numFloors = Integer.parseInt(config.get("floor.number"));
+		int numElevators = Integer.parseInt(config.get("elevator.number"));
+		int multiplier = Integer.parseInt(config.get("time.multiplier"));
+		
+		
+		InetSocketAddress schedulerAddr = null;
+		try {
+			schedulerAddr = new InetSocketAddress(config.get("scheduler.address"), Integer.parseInt(config.get("scheduler.port")));
+		}catch(Exception e) {
+			System.exit(1);
 		}
+		
+		Logger logger = new Logger(true, false, false, false);
+		TimeManagementSystem tms = new TimeManagementSystem(multiplier, logger);
+		
+		ElevatorSubsystem e = new ElevatorSubsystem(schedulerAddr,numElevators,numFloors,logger, tms);
+		//Thread elevatorSubThread = new Thread(e, "ElevatorSubsystemThread");
```

Comment:
Uncomment this out if you feel like it's ok to run right now. If it's still a work in progress don't worry